### DISCUSSION TITLE
Refiners and Loras

### DISCUSSION THREAD

**Discussion by arjundheek** [lllyasviel/Fooocus#3260] [Q&A] [answered]
@mashb1t Hi. Could you assist me how can I load different refiners other than V8 and also how can I load other than lora safetenors1.5 ?

Thank you...

**Comment by mashb1t** [comment, marked as answer]
You can use any model as refiner.
LoRAs should be SDXL.
Download models / LoRAs from https://civitai.com/

**Reply by arjundheek** [reply]
In colab, I see only  juggernautv8 as only available option under refiners. How can I load different refiners into colab?

**Reply by mashb1t** [reply]
- https://github.com/lllyasviel/Fooocus/discussions/1228
- https://github.com/lllyasviel/Fooocus/discussions/3210

**Reply by ILikeToasters** [reply]
Also to add that some models on CivitAI may require you to set up an API key in your account settings and add that to the link to download. The link would be like this if using the API key: `https://civitai.com/api/download/models/000000?token=your_api_key_goes_here`
